### PR TITLE
update profiler

### PR DESCRIPTION
**Pull request by pfitzseb**

### CODE REVIEW

**Comment by aviatesk** [comment]
Do you come up with any solution for that inline highlights can't extend to after the end of line of text ?
image

When the last time I tried to fix this, I remember I found [`TextEditor.decorateMarker`](https://flight-manual.atom.io/api/v1.43.0/TextEditor/#instance-decorateMarker) creates views whose styles can't be tweaked.

**Comment by pfitzseb** [comment]
Fixed that by changing back to the old `overlay` based profile lines.

**Comment by aviatesk** [comment]
Great ... Will take a look again later and let's merge them all then !

File: lib/editor/highlights.coffee [hunk 1]
```diff
@@ -24,15 +24,26 @@ module.exports =
       watch.dispose()
       m.destroy() for m in markers
 
+  profileLineView: (ed, count, classes) ->
+    v = document.createElement 'div'
+    v.classList.add('ink-profile-line')
+    v.classList.add(cl) for cl in classes
+
+    v.style.width = count*ed.preferredLineLength/2 + 'em'
```
Comment (aviatesk) [inline lib/editor/highlights.coffee]: How about
```suggestion
    v.style.width = count*ed.preferredLineLength*ed.defaultCharWidth + 'px'
```
? maybe overkill or fragile in some case ?

**Pull request merged**